src/acquisition/fluview/fluview_update.py

```diff
@@ -132,23 +179,201 @@ def get_ilinet_data(row):
     'n_patients': optional_int(row[14]),
   }
 
+def get_clinical_data(row):
+  if row[0] == 'REGION TYPE' and row != [
+    'REGION TYPE', 
+    'REGION', 
+    'YEAR', 
+    'WEEK', 
+    'TOTAL SPECIMENS', 
+    'TOTAL A',
+    'TOTAL B', 
+    'PERCENT POSITIVE', 
+    'PERCENT A', 
+    'PERCENT B'
+  ]:
+    raise Exception('header row has changed for clinical lab data.')
+  if len(row) == 1 or row[0] == 'REGION TYPE':
+    # this is a header row
+    return None
+  if row[4] == 'X':
+    # data is not reported, ignore this row
+    return None
+  # ignore percentage calculations for now
+  return {
+    'location': fluview_locations.get_location_name(*row[:2]),
+    'epiweek': join_epiweek(int(row[2]), int(row[3])),
+    'total_specimens': int(row[4]),
+    'total_a': optional_int(row[5]),
+    'total_b': optional_int(row[6])
+  }
 
-def load_zipped_csv(filename):
-  """Read rows from a zipped CSV, which is expected to be named ILINet.csv."""
+def get_public_data(row):
+  if row[0] == 'REGION TYPE' and row != [
+    'REGION TYPE', 
+    'REGION', 
+    'SEASON_DESCRIPTION', 
```

Comment:
I'm not familiar with this data, but it's strange to me that there's not an `epiweek` field. Is `epiweek` included in this description somehow?

Reply:
I'm working on this right now, I noticed that there are some inconsistencies in the data. For state-wise public health labs, data comes per season. For hhs, cen, nat, the data comes weekly as expected. I will commit my proposed way of dealing with that in a bit